### PR TITLE
Installation of PyBoolNet from github using pip

### PR DESCRIPTION
**Pull request by carissableker**
Solution to [Issue 16](https://github.com/hklarner/PyBoolNet/issues/16)

Changes:
* setup.py to automatically copy OS dependent dependencies 
* README with updated install instruction

*note version number in README.md is likely wrong*

### CODE REVIEW

**Comment by carissableker** [comment]
https://github.com/hklarner/PyBoolNet/issues/16#issuecomment-654905243: 
> Thinking a bit more about it, it would be even better for me if setup.py could first check if the CONDA_BUILD environment variable is set to skip the copy of dependencies when it is, instead of removing them afterwards.
> 
> Once these changes are in, I will add a pull request to include the conda recipe


@aurelien-naldi @hklarner Would you like me to update my pull request? I am not experienced in conda-build.

**Comment by aurelien-naldi** [comment]
If you don't use conda yourself, don't bother with it for now. When your PR gets merged, I can make another one to adapt setup.py and add the conda recipe.

**Comment by hklarner** [comment]
@carissableker I made two comments in the PR. Did you see them?

**Comment by carissableker** [comment]
I am sorry, I have not seen any comments or received any notifications regarding this PR. Could you send me a link to the comments?

File: setup.py [hunk 1]
```diff
@@ -20,7 +38,7 @@
     package_data_files.extend([os.path.join(root, x) for x in filenames])
 
 setup(name="PyBoolNet",
-      version="2.2.8",
+      version="2.2.8.a",
```
[the remark this answers is not in the document]
Comment (carissableker) [reply setup.py:41]: This is the correct argument for the `setuptools` version of `setup`. You previously had the `setup` function from ` distutils.core`. As far as I can tell, setuptools is currently preferred to distutils:  [stackoverflow discussion](https://stackoverflow.com/a/14753678) and [Python packaging guide](https://packaging.python.org/tutorials/packaging-projects/).

File: setup.py [hunk 2]
```diff
@@ -40,5 +58,5 @@
           "Natural Language :: English",
           "Topic :: Scientific/Engineering :: Bio-Informatics",
       ],
-      requires=['networkx (==1.11)'],
+      install_requires=['networkx (==1.11)']
```
[the remark this answers is not in the document]
Comment (carissableker) [reply setup.py:61]: This is the correct argument for the `setuptools` version of `setup`. You previously had the `setup` function from ` distutils.core`. As far as I can tell, setuptools is currently preferred to distutils:  [stackoverflow discussion](https://stackoverflow.com/a/14753678) and [Python packaging guide](https://packaging.python.org/tutorials/packaging-projects/).

File: README.md [hunk 3]
```diff
@@ -16,7 +16,17 @@ For bug reports and feedback do not hesitate to open issues at [PyBoolNet issues
...
-Do not try to install this git repository directly, see [Issue #16](https://github.com/hklarner/PyBoolNet/issues/16). Instead download the latest release from https://github.com/hklarner/PyBoolNet/releases and use pip to install. For example:
+To install directionly from this repository use:
+
+```bash 
+pip install git+https://github.com/hklarner/PyBoolNet
+```
+To install a particular release, append the version number to the end of the github url with an `@`. E.g.: 
+
+```bash 
+pip install git+https://github.com/hklarner/PyBoolNet@2.2.9
```
Comment (carissableker) [inline README.md:27]: Need to create a release 2.2.9 on github.

Comment (hklarner) [reply README.md:27]: you mean a tag, right?

**Pull request merged**